code/modules/reagents/reagent_containers/cups/bottle.dm

```diff
@@ -7,8 +7,8 @@
 	fill_icon_state = "bottle"
 	inhand_icon_state = "atoxinbottle"
 	worn_icon_state = "bottle"
-	possible_transfer_amounts = list(5, 10, 15, 25, 30)
-	volume = 30
+	possible_transfer_amounts = list(5, 10, 15, 25, 50)
+	volume = 50
```

Comment:
The point is to make all bottles have max volume of 50, like beakers. I didn't plan to update any subtypes and default reagents. Multiver bottle will now spawn with 30/50 instead of 30/30 units.